package.json

```diff
@@ -30,6 +30,7 @@
   },
   "types": "./types/fh-sync.d.ts",
   "scripts": {
+    "example": "cd examples/basic-express-example/ && npm i && node server.js",
```

Comment:
I think this is useful to save people the effort of doing that. Similar to a 'grunt serve', but thankfully without the 'grunt' 😉 

Usually I think examples in most repositories don't require having to install dependencies in a nested folder a second time either? 

My main thought was to make things easy as pososble. I can make this change, but some food for thought!